controllers/validator.go

```diff
@@ -23,6 +23,9 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 		return false, errors.New("DPA CR Velero configuration cannot be nil")
 	}
 
+	if dpa.Spec.Configuration.Velero.Args.RestoreResourcePriorities != "" {
```

Comment:
I think: ` == ""` if it is `!= ""` then it would ONLY overwrite someone's set priorities I think?